### ISSUE TITLE
[FEATURE] Add PWA (Progressive web app) in vibey

### ISSUE THREAD

**Issue by SubhamB2003** [UniKonf/Vibey#587]
### Detailed description

Hey, I want to the PWA to this. It's a very good feature and also we can install on our phone or desktop.
Can you please assign it to me?

### Context

.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Deepu178** [comment]
@SubhamB2003 how will you implement it? Can you explain it in details?

**Comment by SubhamB2003** [comment]
Yeah, I installed the package next-pwa it's required to install for setup pwa. After setup the things you have an option to install it.

Like this
![image](https://github.com/UniKonf/vibey/assets/94003231/33471b39-a8ba-4dee-9593-ccefc7719d31)

**Comment by SubhamB2003** [comment]
@Deepu178 Can you please check
